fix(item): add a default image url

codeyourfaceoff opened the pull request:
in order to not force users to have to upload when creating a new item, adding a default image to the Item model is the best solution.

Providing a value for the image preview in the component when an image is not on the item itself allows us to always have an image to display, while at the same time not taking up space in the database. As well as if we change our mind on the default image's url in the future, it's a 1 line change instead of a database update.

#INC-5312: Missing image
Fixes https://github.com/ObelusFamily/Anythink-Market-3yia1/issues/2

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

vanessa-cooper commented:
Thanks codeyourfaceoff, I'll review the changes right away!

vanessa-cooper reviewed (changes requested):
Make sure all checks pass, only then you can merge the PR.

codeyourfaceoff committed:
fix(item): add a default image url

in order to not force users to have to upload when creating a new item, adding a default image to the Item model is the best solution.

Providing a value for the image preview in the component when an image is not on the item itself allows us to always have an image to display, while at the same time not taking up space in the database. As well as if we change our mind on the default image's url in the future, it's a 1 line change instead of a database update.

#INC-5312: Missing image
Fixes #2

codeyourfaceoff committed:
fix(lint): format code style

vanessa-cooper reviewed (approved):
Excellent job codeyourfaceoff! You're now able to merge the PR.
